portfolio/src/main/webapp/index.html

```diff
@@ -34,6 +34,16 @@ <h3>People reading my intro:</h3>
       <p>Click here to get a random fact about me:</p>
       <button onclick="addRandomFact()">Fact</button>
       <div id="fact-container"></div>
+
+      <form action="/data" method="POST">
+
+        <p>Type in a comment:</p>
+        <textarea name="comment-input"></textarea>
+
+        <input type="submit" />
```

Comment:
Optional: you could a value="Submit" tag or something here. It seems the default value takes the browser language, so this button appears in a different language that the "Fact" button above.